### PR TITLE
fixed customisable date bug

### PR DESCRIPTION
**Pull request by Codebmk**
#### Summary of Changes (What does this PR do?)
- Fixed customisable chart start date so it reflects the actual start date and not the day before the start date

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [ ] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Do a git fetch and check out to this branch locally

#### What are the relevant tickets?
- [ticket_id]()

### CODE REVIEW

**Review by 123MwanjeMike — APPROVED** [review]
LGTM

**Review by Baalmart — COMMENTED** [review]
Hi @Codebmk 
For some reason, I am unable to run this PR....all the chats are failed to load and on adding a chat, I get some undescriptive errors.

Tried to switch from staging to production but same thing still. Please check out the attached console logs....

Screenshot 2022-03-10 at 15 17 09
Screenshot 2022-03-10 at 15 18 02

**Comment by Codebmk** [comment]
> Hi @Codebmk For some reason, I am unable to run this PR....all the chats are failed to load and on adding a chat, I get some undescriptive errors.
> 
> Tried to switch from staging to production but same thing still. Please check out the attached console logs....
> 
> Screenshot 2022-03-10 at 15 17 09
> 
> Screenshot 2022-03-10 at 15 18 02

@Baalmart i don't have those bugs when i run the application on my end🤯 could you try to run the PR once more and see if the bugs persist?

File: netmanager/src/views/components/d3/CustomisableChart.js [hunk 1]
```diff
@@ -170,12 +171,8 @@ const CustomisableChart = (props) => {
...
       return [new Date(period.startDate), new Date(period.endDate)];
     }
-    let endDate = period.endDate ? new Date(period.endDate) : new Date();
-    let startDate = new Date(
-      endDate.getFullYear(),
-      endDate.getMonth(),
-      endDate.getDate() - period.unitValue
-    );
+    let endDate = new Date();
+    let startDate = moment().subtract(period.unitValue, 'days');
```
Comment (ja-odur) [inline netmanager/src/views/components/d3/CustomisableChart.js:175]: @Codebmk Please set the unit from period object (period.unit). The unit could be months and days.

**Review by Baalmart — COMMENTED** [review]
Thanks @Codebmk I have been able to run, successfully using staging. Just a few clarifications I need....

My charts are taking forever to load! Any thoughts? Please see attached screenshots.
Screenshot 2022-03-11 at 11 00 25
Screenshot 2022-03-11 at 11 01 26

**Review by Baalmart — APPROVED** [review]
All LGTM to me. I hope @ja-odur 's review comment was addressed? @Codebmk

**Pull request merged**